docs/_content/standaard/catalogi/index.md

```diff
@@ -118,6 +118,28 @@ parametrisering is aan validatieregels onderhevig:
     * MOET een waarde hebben indien de afleidingswijze `termijn` is
     * MAG GEEN waarde hebben in de andere gevallen
 
+#### Concepten
+
+De resources `Zaaktype`, `InformatieObjecttype` en `Besluittype` bevatten het veld `concept`, indien dit veld aangemerkt is als `True`, dan betreft het een niet-definitieve versie van het object die niet buiten de Catalogi API gebruikt mag worden.
+
+Om het object te veranderen naar een definitieve versie, kan het object via de API gepubliceerd worden. Zodra een object gepubliceerd is, staat het veld `concept` op `False`.
```

Comment:
```suggestion
Om de versie van een object definitief te maken ("publiceren"), bestaat er een `publish` operatie.
Dit is de tegenhanger van het attribuut `concept`, dus na publiceren heeft `concept` de waarde `false`.
```

@michielverhoef ik snap je bezwaar, omdat het tegenhangers zijn van elkaar. Echter, de actie zelf `publish` noemen (of dus in nederlands `publiceer`, deze hebben over het hoofd gezien) dekt de lading van de actie het best.

De reden voor het `concept` veld is dat er gefilterd kan worden op drie semantische waardes: `status=alles`, `status=concept` of `status=definitief`. Je zit dus met een boolean op een object (wel of niet concept/gepubliceerd) die zich vertaalt in 3 mogelijke status waardes voor de lijst. Die derde waarde is nodig voor de filters omdat je anders niet de default van "toon enkel definitieve versies" netjes kan toegankelijk maken (en tevens de optie bieden om _alles_ op te vragen).